Code/Editor/CryEdit.cpp

```diff
@@ -598,13 +598,24 @@ void CCryEditApp::OnFileSave()
 
     const QScopedValueRollback<bool> rollback(m_savingLevel, true);
 
-    auto* prefabIntegrationInterface = AZ::Interface<AzToolsFramework::Prefab::PrefabIntegrationInterface>::Get();
-    AZ_Assert(prefabIntegrationInterface != nullptr, "PrefabIntegrationInterface is not found.");
+    bool usePrefabSystemForLevels = false;
+    AzFramework::ApplicationRequests::Bus::BroadcastResult(
+        usePrefabSystemForLevels, &AzFramework::ApplicationRequests::IsPrefabSystemEnabled);
 
-    prefabIntegrationInterface->SaveCurrentPrefab();
+    if (!usePrefabSystemForLevels)
+    {
+        GetIEditor()->GetDocument()->DoFileSave();
+    }
+    else
+    {
+        auto* prefabIntegrationInterface = AZ::Interface<AzToolsFramework::Prefab::PrefabIntegrationInterface>::Get();
+        AZ_Assert(prefabIntegrationInterface != nullptr, "PrefabIntegrationInterface is not found.");
 
-    // when attempting to save, update the last known location using the active camera transform
-    AzToolsFramework::StoreViewBookmarkLastKnownLocationFromActiveCamera();
+        prefabIntegrationInterface->SaveCurrentPrefab();
+
+        // when attempting to save, update the last known location using the active camera transform
+        AzToolsFramework::StoreViewBookmarkLastKnownLocationFromActiveCamera();
+    }
 }
```

Comment:
I'm not sure if there was an incorrect merge, but there seem to be a lot of references to `IsPrefabSystemEnabled` being reinstated which were removed in `development` by this PR: https://github.com/o3de/o3de/pull/17657

`AzFramework::ApplicationRequests::IsPrefabSystemEnabled` is always going to return `true` now, so the `else` code branches are unused.

Reply:
Perhaps a rebase on latest development is needed in this PR.

Reply:
yes, just fixed it. didn't notice.